js/app/components/live-dashboard/stream-key.tsx

```diff
@@ -65,20 +65,20 @@ export default function StreamKeyScreen() {
           <Button
             marginHorizontal={10}
             backgroundColor={
-              protocol === "whip" ? "$accentBackground" : "$grey2"
+              protocol === "rtmp" ? "$accentBackground" : "$grey2"
             }
-            onPress={() => setProtocol("whip")}
+            onPress={() => setProtocol("rtmp")}
           >
-            WHIP
+            RTMP (beta, preferred)
```

Comment:
looooooooooool
I think let's just go with
```suggestion
            RTMP
```